New version: InternalFluidFlow v2.0.0

JuliaRegistrator opened the pull request:
- Registering package: InternalFluidFlow
- Repository: https://github.com/aumpierre-unb/InternalFluidFlow.jl
- Created by: @aumpierre-unb
- Version: v2.0.0
- Commit: 154307fd6e9ce57f62afa640e0dd5e737a7fc2b8
- Reviewed by: @aumpierre-unb
- Reference: https://github.com/aumpierre-unb/InternalFluidFlow.jl/commit/154307fd6e9ce57f62afa640e0dd5e737a7fc2b8#commitcomment-146833668
- Description: A toolbox for internal fluid flow for Julia
<!-- bf0c69308befbd3ccf2cc956ac8a46712550b79fc9bfb5e4edf8f833f05f4c18b06eddad8845b45beb9f45c2b8020dd62ab3c37d78b99cd14e4388f4bf4529d14645e88d3320d43adec24e87ea3f7b43a5aa108d97a81c791df39832b9990b7cbfc964c72eb09824cc4620a274d0447919e70c496b8121aa349351b42f3c2ac94a1f6ddc408705c4b36c91e0528a2f3d75585759bf2f1bb5661e62bfe50a3f998eb560ce3c91165f72288e887111cf758177448d370d6cac4d2fbd88d9af6d3657494362a800e72a162a8b04b315db2c71fbbef176eb4d40df24f8e049f36de10ec3e29641bc569b4c2c68abf677f5cf -->

aumpierre-unb commented:
@JuliaRegistrator register
[no block]

aumpierre-unb commented:
@JuliaRegistrator register
[noblock]

aumpierre-unb commented:
This is a major release with a few important changes in mais and internal functions. The algorithms, the functions' returns, Moody diagram and the warning messages have been upgraded.
[noblock]

JuliaRegistrator committed:
New version: InternalFluidFlow v2.0.0

UUID: a3f4ffcb-63ab-449f-b58f-50e21d5ccbd2
Repo: https://github.com/aumpierre-unb/InternalFluidFlow.jl.git
Tree: 4594c7ed49a6b7f4ce7f41ff435ca3a37cfdd88d

Registrator tree SHA: 17aec322677d9b81cdd6b9b9236b09a3f1374c6a

aumpierre-unb commented:
### InternalFluidFlow 2.0.0 (2024-09-17)
* Main and internal functions reformulated
* All functions now scan for solutions within the laminar (Re < 4e3) and the turbulent (Re > 2.3e3) bounds
* D, v, and Q are now keyword arguments in h2fRe function
* f is a keyword arguments in f2Re function
* Re is a keyword arguments in Re2f function
* h2fRe function now accepts pipe roughness
* Computed values now are presented as structure Moody(Re, f, ε)
* Moody diagrams now display more turbulent lines, with relative roughness identified at the right margin
* doPlot function now is available to plot schematic Moody diagram
* Tests are extended
[noblock]

aumpierre-unb commented:
[merge approved]
